config/ember-try.js

```diff
@@ -1,46 +1,25 @@
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
 module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-2.18',
```

Comment:
The new scenarios for Ember 3.16, 3.20 and Embroider should be added without replacing the existing scenarios. We need them to ensure that we do not break compatibility commitment without noticing. Maybe we should add a Ember 3.12 scenario as well to cover all LTS versions since 2.18.

Reply:
I fear I got confused. The addon does not seem to run Ember Try at all. It uses a custom setup to test against multiple Ember CLI versions: https://github.com/rwjblue/ember-cli-content-security-policy/blob/master/.github/workflows/ci.yml#L145

This implicitly covers ember-source versions as well. But does not include ember-source@2.18.

It also means that Embroider tests are not run yet. Even after merging this one.

I guess we need to revisit the compatibility guarantee and test suite. But it could be done independently from this merge request.